backend/src/firebase/admin/admin.service.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable prettier/prettier */
+import { Injectable, OnModuleInit } from '@nestjs/common';
+import * as admin from 'firebase-admin';
+import * as serviceAccount from '../../config/genealogicaltree-6e0e2-firebase-adminsdk-74kt0-e5f4045782.json';
+
+@Injectable()
+export class AdminService implements OnModuleInit {
+  onModuleInit() {
+    admin.initializeApp({
+      credential: admin.credential.cert(serviceAccount as admin.ServiceAccount),
+    });
+  }
+
+  getAuth() {
```

Comment:
Dodajemy typy zwrotek i argumentów we wszystkich funkcjach, to jest typescript :)

Reply:
To mam już zrobione inaczej, mam:
private firestore: FirebaseFirestore.Firestore;
private auth: admin.auth.Auth;

Reply:
Elegancko